sdk/state/open.go

```diff
@@ -9,11 +9,27 @@ import (
 	"github.com/stellar/go/xdr"
 )
 
+type Trustline = txbuild.Trustline
```

Comment:
I'm not tied to this name. `Trustline` makes sense now but might make less sense after adding to the Close transactions 🤔 

Might be more obvious what a better name is when it's added then

Reply:
I think TrustLine is the most accurate term. The users are agreeing on what TrustLine the escrow accounts must have, so it's perfect in my mind. It's actually supposed to be two words, Trust Line, therefore TrustLine but I guess the Go SDK didn't follow the protocol.